Create meta helper

martinbean opened the pull request:
Added a helper to create simple key–value pair meta tags. Also created a corresponding test.

adamgoose commented:
Can you please add the ability to pass a third argument `array $attributes = []` that can be merged with the `compact('name', 'content')`? This will make it easier to install Facebook OpenGraph Meta Tags, which do not use the conventional name/content attributes (dumb, right?).

martinbean commented:
@adamgoose Sure. Neither does schema.org mark-up, which use `itemprop` and `content` as well ;)

martinbean commented:
@adamgoose Added `$attributes` as an optional third parameter and squashed the commit. Also added an additional test to cover creating an Open Graph meta tag.

adamgoose commented:
Sorry, but it should use `array_merge()`. You'd be surprised how many people will put a `name` or `content` key in the `array $attributes`. Not sure how the + syntax will handle that.

martinbean commented:
@adamgoose Oh. I took that approach from one of the other methods (not sure which one as on phone).

I’ll take a look later this evening or tomorrow morning as I’m travelling home, and heading straight out for the evening.

martinbean commented:
@adamgoose Just pushed an update using `array_merge`. Let me know if that’s more what you’re after.

RobinMalfait commented:
@martinbean failing tests :-1:

martinbean commented:
@RobinMalfait Fixed.

JosephSilber commented on tests/HtmlBuilderTest.php:
```diff
@@ -10,36 +10,49 @@
+	public function setUp()
+	{
+		$this->urlGenerator = new UrlGenerator(new RouteCollection, Request::create('/foo', 'GET'));
+		$this->htmlBuilder	= new HtmlBuilder($this->urlGenerator);
```
This should be two spaces.

martinbean replied:
@JosephSilber Not sure where you mean. Do you mean in between `$this->htmlBuilder` and HtmlBuilder($this->urlGenerator);`= new`?

tshafer replied:
@martinbean Yes, please simply clean up the formatting on this.

martinbean commented:
@JosephSilber @tshafer Indentation in test fixed. Sorry it took so long to address it, I was away for a few days last week.

tshafer commented:
Thanks!

martinbean commented:
@tshafer My pleasure!
